features/developer_portal/login.feature

```diff
@@ -4,12 +4,49 @@ Feature: Login feature
 
   Background:
     Given a provider "foo.3scale.localhost"
-      And provider "foo.3scale.localhost" has multiple applications enabled
-      And a buyer "bob" signed up to provider "foo.3scale.localhost"
+    And provider "foo.3scale.localhost" has multiple applications enabled
+    And a buyer "bob" signed up to provider "foo.3scale.localhost"
+    And the current domain is "foo.3scale.localhost"
 
   @security
   Scenario: Buyer can log in with csrf protection enabled
-    Given the current domain is foo.3scale.localhost
     When I go to the login page
-     And I fill in the "bob" login data
+    And I fill in the "bob" login data
+    Then I should be logged in the Development Portal
```

Comment:
how do we know csrf is enabled?

Reply:
I'm gonna sound silly but why is CSRF relevant in the context of captcha? 😅

Reply:
Scenarion name is `Buyer can log in with csrf protection enabled`. That's why I asked about CSRF. Perhaps scenario name needs to be changed? Or CSRF part made more obvious.

Reply:
I removed this, it didn't make sense IMO